apps/src/templates/codeReviewGroups/CodeReviewGroup.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import FontAwesome from '../FontAwesome';
+import color from '@cdo/apps/util/color';
+import StudentGroup from './StudentGroup';
+
+export default function CodeReviewGroup({
+  droppableId,
+  members,
+  name,
+  onNameUpdate,
+  onDelete
+}) {
+  const handleNameUpdate = event => {
+    onNameUpdate(droppableId, event.target.value);
+  };
+  return (
+    <div style={styles.groupContainer}>
+      <div style={styles.headerContainer}>
+        <input
+          value={name}
+          style={styles.nameInput}
+          onChange={handleNameUpdate}
+          placeholder={i18n.enterGroupName()}
+        />
+        <button
+          style={styles.deleteButtonContainer}
+          onClick={() => onDelete(droppableId)}
+          type={'button'}
+          aria-label={i18n.deleteGroup()}
+        >
+          <FontAwesome icon={'trash'} style={styles.deleteButton} />
+        </button>
```

Comment:
Could this just be a `Button` component with no text (only icon)?

Reply:
Looks like `Button` has text as a required prop -- if I try to just provide a blank string, it doesn't look great. I don't see other examples in our code base where we do this either, so inclined to leave as-is. It does seem like there's probably a more standard way to do this though...